ports/mimxrt/boards/MIMXRT1060_EVK/mpconfigboard.mk

```diff
@@ -0,0 +1,7 @@
+MCU_SERIES = MIMXRT1062
+MCU_VARIANT = MIMXRT1062DVJ6A
+
+JLINK_PATH = /media/RT1060-EVK/
```

Comment:
Perhaps this should use `?=` so it can be overridden?

Reply:
Thanks for your comments, I have updated the PR.